tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -33,9 +34,9 @@ import (
 )
 
 const (
-	iamPolicyS3FullAccess     = "arn:aws:iam::aws:policy/AmazonS3FullAccess"
-	iamPolicyS3ReadOnlyAccess = "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess"
-	iamPolicyS3NoAccess       = "arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess" // `AmazonEC2ReadOnlyAccess` gives no S3 access
+	iamPolicyS3FullAccess     = "AmazonS3FullAccess"
+	iamPolicyS3ReadOnlyAccess = "AmazonS3ReadOnlyAccess"
+	iamPolicyS3NoAccess       = "AmazonEC2ReadOnlyAccess" // `AmazonEC2ReadOnlyAccess` gives no S3 access
```

Comment:
There is also partition hardcoded in this file: https://github.com/awslabs/mountpoint-s3-csi-driver/blob/main/tests/e2e-kubernetes/scripts/eksctl-patch.json

Do we need to adjust it too?

Reply:
Possibly - I can't remember how I created my test China cluster